src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC_InitRefIndexPage.js

```diff
@@ -304,10 +361,16 @@ class CJ4_FMC_InitRefIndexPage {
         ]);
         fmc.updateSideButtonActiveStatus();
     }
-    static calcDistance(currentDistance, currentWaypoint, newWaypoint) {
-        let waypointDistance = Avionics.Utils.computeDistance(currentWaypoint.infos.coordinates, newWaypoint.infos.coordinates);
-        return currentDistance + waypointDistance;
-    }
+    //static calcDistance(currentDistance, currentWaypoint, newWaypoint) {
```

Comment:
Do we need to delete this commented out stuff?

Reply:
Yea, I guess since it is saved in the commit, you said I could delete it and not lose it forever, so I'll do that.